projects/storefrontlib/cms-components/content/tab/panel/tab-panel.component.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, Input } from '@angular/core';
+import { Tab, TAB_MODE } from '../tab.model';
+
+@Component({
+  selector: 'cx-tab-panel',
+  templateUrl: './tab-panel.component.html',
+})
```

Comment:
```suggestion
changeDetection: ChangeDetectionStrategy.OnPush // Optimizes performance by reducing unnecessary checks
```

Reply:
OnPush change detection causes issues with responsiveness when switching between Tab and Accordian modes. Tests fail as a result.